cmd/sandbox/exec.go

```diff
@@ -0,0 +1,45 @@
+package sandbox
+
+import (
+	"context"
+	"fmt"
+
+	cmdCore "github.com/flyteorg/flytectl/cmd/core"
+	"github.com/flyteorg/flytectl/pkg/docker"
+)
+
+const (
+	execShort = "Execute any command in sandbox"
+	execLong  = `
+Execute command will run any command in sandbox.
```

Comment:
Will run non-interactive commands and return immediately with the output?

Reply:
yes

Reply:
I believe Ketan meant to ask you to update the text to say that...